src/DssSpell.t.sol

```diff
@@ -568,45 +578,32 @@ contract DssSpellTest is DssSpellTestBase {
     struct Payee {
         address token;
         address addr;
-        uint256 amount;
+        int256 amount;
     }
 
     struct PaymentAmounts {
-        uint256 dai;
-        uint256 mkr;
-        uint256 usds;
-        uint256 sky;
+        int256 dai;
+        int256 mkr;
+        int256 usds;
+        int256 sky;
```

Comment:
Why do we need to use `int256` here?

Reply:
The changes to the payment tests were made to account for the source of the payment. Last spell we used to convert MKR into SKY and DAI into USDS (which might lower total supply of some tokens) while this time we're minting tokens (so it will increase total supply). To account for both cases, I've switched to signed integers

Reply:
Makes sense :100: